content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,32 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for Contributing content
+
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects ([Graduated/Incubating](https://www.cncf.io/projects/), [Sandbox](https://www.cncf.io/sandbox-projects/)).
+The CNCF doesn't maintain a list of archived projects. However, you can
+ search the CNCF Technical Oversight Committee GitHub 
+ [pull requests](https://github.com/cncf/toc/labels/archive) to find proposals to archive projects.
+
+Kubernetes documentation does not host duplicate content sourced from
+ multiple locations ("dual-sourced" content).
+
+Examples of content that is and is not allowed:
+
+{{< table caption="Examples of content that is and is not allowed" >}}
+| Allowed |                           Action                                              |                                                    Reason                                                   |
+|:-------:|:------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+|    Yes  | Linking to Kubernetes in Docker (KinD) [docs](https://kind.sigs.k8s.io/docs/user/quick-start) from k8s docs | The KinD [project](https://github.com/kubernetes-sigs/kind) resides in `kubernetes-sigs` GitHub organization.                                                     |
+|    No   | Embedding/hosting KinD documentation in the k8s website    | Dual-sourced content requires duplicated effort from project maintainers and tends to go stale/rotten more quickly. |
```

Comment:
Line 59: For the `Reason`: Could just say, Dual-sourced content.
nit: Move the explanation and definition about dual-sourced content to the second paragraph above

Reply:
@kbhawkey good suggestion! Re-worded.